modules/manage/pages/cluster-maintenance/configure-availability.adoc

```diff
@@ -5,6 +5,29 @@
 
 Optimize the availability of your clusters by configuring and tuning properties.
 
+// Don't display ListKafkaConnections in Cloud docs until support is added
+ifdef::env-cloud[]
+TIP: Before you configure connection limits or reconnection settings, start by gathering detailed data about your client connections. Use the xref:reference:public-metrics-reference.adoc#redpanda_rpc_active_connections[`redpanda_rpc_active_connections` metric] to view current Kafka client connections.
+endif::[]
+ifndef::env-cloud[]
+[TIP]
+====
+Before you configure connection limits or reconnection settings, start by gathering detailed data about your client connections. 
+
+* Internal metrics that follow the `vectorized_kafka_rpc_.*connect.*` naming pattern provide details on Kafka client connection activity. For example, xref:reference:internal-metrics-reference.adoc#vectorized_kafka_rpc_active_connections[`vectorized_kafka_rpc_active_connections`] reports the current number of active connections.
+* For Redpanda version 25.3 and later, use the `rpk cluster connections` command or the Admin API v2 ListKafkaConnections endpoint to identify on a per-broker basis:
++
+-- 
+** Which clients and applications are connected
```

Comment:
@kbatuigas This is the punchline, but having trouble understanding

-  why this is in the 'availability page' and not 'manage throughput'
- where are the examples of rpk?

Reply:
@mattschumpert -

- URL changed to configure-client-connections
- The idea is that (based on my understanding) ListKafkaConnections answers many different types of questions related to client connections, not just throughput. So the tip on this page provides slightly more general guidance on what data you can get from the API. I'm also updating the [Manage Throughput doc](https://github.com/redpanda-data/docs/pull/1389/commits/24eacfb6a42e4520a6efbf1bc0ce684cd27ab183#diff-054298b57d90b7ce401e83a4c630c53ea43885dd92d67c79a3fe8ab93fe618e5R20) to specifically show a throughput example
- rpk is not available yet and the latest guidance I got is that the specific features and flags are still TBD